packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -440,13 +440,23 @@ export const transformers = (model: ModelData): string[] => {
 	}
 
 	if (model.pipeline_tag && LIBRARY_TASK_MAPPING.transformers?.includes(model.pipeline_tag)) {
-		const pipelineSnippet = [
-			"# Use a pipeline as a high-level helper",
-			"from transformers import pipeline",
-			"",
+		let pipelineSnippet = ["# Use a pipeline as a high-level helper", "from transformers import pipeline", ""];
+
+		if (model.tags?.includes("conversational")) {
+			pipelineSnippet = pipelineSnippet.concat([
+				"messages = [",
+				'    {"role": "user", "content": "Who are you?"},',
+				"]",
+			]);
+		}
+		pipelineSnippet = pipelineSnippet.concat([
 			`pipe = pipeline("${model.pipeline_tag}", model="${model.id}"` + remote_code_snippet + ")",
-		].join("\n");
-		return [pipelineSnippet, autoSnippet];
+		]);
+		if (model.tags?.includes("conversational")) {
+			pipelineSnippet = pipelineSnippet.concat(["pipe(messages)"]);
+		}
+
+		return [pipelineSnippet.join("\n"), autoSnippet];
```

Comment:
In another PR we could use `apply_chat_template` and call `generate` in the `autoSnippet` for conversational models.